src/main/models/wallet-setting.js

```diff
@@ -44,7 +44,7 @@ class WalletSetting extends BaseModel {
 				showDesktopNotifications: { type: 'integer' },
 				previousTransactionCount: { type: 'integer' },
 				txHistoryLastSyncedBlock: { type: 'integer' },
-				airDropCode: { type: 'integer' }
+				airDropCode: { type: ['integer', 'null'] }
```

Comment:
@tmchedlishvili you said it should be string, no?

Reply:
@sk91 For now I don't know it exactly, it should be nullable definitely, so i added it.